taca/utils/bioinfo_tab.py

```diff
@@ -147,46 +149,41 @@ def get_ss_projects(run_dir):
     FCID = run_name_components[3][1:]
     newData = False
     miseq = False
-    
-    xten_samplesheets_dir = os.path.join(CONFIG['bioinfo_tab']['xten_samplesheets'],
-                                    current_year)
-    hiseq_samplesheets_dir = os.path.join(CONFIG['bioinfo_tab']['hiseq_samplesheets'],
-                                    current_year)
-    #If it is not hiseq
-    FCID_samplesheet_origin = os.path.join(hiseq_samplesheets_dir, '{}.csv'.format(FCID))
-    if not os.path.exists(FCID_samplesheet_origin):
-        #If it is not xten
-        FCID_samplesheet_origin = os.path.join(xten_samplesheets_dir, '{}.csv'.format(FCID))
-        if not os.path.exists(FCID_samplesheet_origin):
-            #It is miseq
-            miseq = True
-            lanes = str(1)
-            #Pattern is a bit more rigid since we're no longer also checking for lanes
-            sample_proj_pattern=re.compile("^((P[0-9]{3,5})_[0-9]{3,5})$")
-            
-            FCID_samplesheet_origin = os.path.join(run_dir,'Data','Intensities','BaseCalls', 'SampleSheet.csv')
-            if not os.path.exists(FCID_samplesheet_origin):
-                FCID_samplesheet_origin = os.path.join(run_dir,'SampleSheet.csv')
-                if not os.path.exists(FCID_samplesheet_origin):
-                    logger.warn("Cannot locate the samplesheet for run {}".format(run_dir))
-                    return ['UNKNOWN']
 
-        try:
-            ss_reader=SampleSheetParser(FCID_samplesheet_origin)
-            data=ss_reader.data
-        except:
-            logger.warn("Cannot initialize SampleSheetParser for {}. Most likely due to poor comma separation".format(run_dir))
-            return []
-        if 'Description' in ss_reader.header and ss_reader.header['Description'] not in ['Production', 'Applications']:
-            logger.warn("Run {} detected as a non platform MiSeq run. Disregarding it.".format(run_dir))
-            return []
-    else:
+    #Miseq case
+    if 'miseq' in run_dir:
```

Comment:
I do not like making this assumtions... this means we hope that the path contains miseq... we have the run_id so we can understand what kind of FC is (HiSeq, MiSeq, or HiSeqX).